lua/luasnip/nodes/util.lua

```diff
@@ -851,6 +863,8 @@ local function collect_dependents(node, which, static)
 	return tbl_util.set_to_list(dependents_set)
 end
 
+---@param args (string[])?
```

Comment:
`args` should be `(string|SnippetString)[]` I think

Reply:
No? I can only find calls a list of string :thinking:

Reply:
Ahh yeah I see :D
The fix looks good :+1: